build/yamls/flow-collector/logstash/logstash.conf

```diff
@@ -0,0 +1,50 @@
+input {
```

Comment:
Could we add some comments to explain the configurations?

Reply:
Done